content/docs/operating/storage.md

```diff
@@ -86,3 +86,12 @@ storage directory:
    1. Stop Prometheus.
    1. `rm -r <storage path>/*`
    1. Start Prometheus.
+
+## Hash collisions
+
+Prometheus uses 64-bit fingerprints to identify time series. On a
```

Comment:
The rule of thumb for the twin paradox is that the probability of collision goes over .5 when you've got sqrt(max size) elements.

So for a space 2^64, we'd be looking at series problems at 2^32. Thus 10^6 ~= 2^20 should be okay from rough math - but the chances of hardware/software failure is much smaller than 2^12.